index.html

```diff
@@ -3356,6 +3356,31 @@ <h3><em>RTL/bidi text (TBD)</em></h3>
 
 
 
+
+
+
+
+<section id="typ_box_posn" class="subtopic">
+<h3>Setting box positioning coordinates when text direction varies</h3>
+
+  	<div class="req" id="vertical_box_posn">
+	<p class="advisement">Box positioning coordinates must take into account whether the text is horizontal or vertical.</p>
+	</div>
+
+<p>It is typical, when localizing a user interface or web page, to create mirror-images for the RTL and LTR versions. For example, it is likely that a box that appears near the left side of a window containing English content would appear near the right side of the window if the content is Arabic or Hebrew. It should preferably automatic for this to change, based on the base direction of the current context, unless there is a strong reason for using absolute geometry. One way to achieve this is to use keywords such as <code class="kw" translate="no">start</code> and <code class="kw" translate="no">end</code>, rather than <code class="kw" translate="no">left</code> and <code class="kw" translate="no">right</code>, to indicate position.</p>
+  </section>
+
+
+
+<section id="typ_logical" class="subtopic">
+<h3><em>Logical properties (TBD)</em></h3>
```

Comment:
Why are we adding this as a subsection? Isn't logical vs. physical already embodied by some of our BPs?

Reply:
We talk about directional metadata in earlier sections, but this is in a section related to typographic support and is a placeholder for things that should say 'allow your styling to be defined using logical properties'.

Reply:
I'm still not sure I understand. Do you mean thinks like "use `<em>` instead of `<i>` and `<strong>` instead of `<b>` when emphasizing text"?

Reply:
No, i mean something like:

**Specifications that define placement of typographic items should allow the user to work with logical property values (such as -start, -end, etc.) rather than only defining fixed positions (eg. -left, -right).**

Reply:
That's what I originally understood. 

I think we should add a paragraph of description--and include the BP you just mooted in your comment!

I would also change the section `id`. Lots of things are "logical" 😸 or even "logical-vs-physical"/"logical-vs-visual". Perhaps `typ_logical_pos` (_pos_ for "positioning")??